frontend/src/metabase/visualizations/components/skeletons/util/animation.ts

```diff
@@ -14,13 +14,6 @@ export const fadingKeyframes = keyframes`
     opacity: 0.5;
   }
 `;
-
-export const containerStyles = css`
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-`;
-
 export const animationStyles = css`
```

Comment:
I don't think that css styles should go to the utils folder. By looking at the code it seems that we could either:
- extract `SkeletonImage` as a styled component in its own folder and use that instead of `styled.svg` + `animationStyles` pair
- do not change the components but place these styles in the corresponding `*.styled.tsx` file in one of the component folders (presumably `VisualizationSkeleton` or `ChartSkeleton`). <- I'd probably do this

Reply:
Did the second option 😄